modular_skyrat/modules/company_imports/code/armament_datums/kz_frontier.dm

```diff
@@ -0,0 +1,70 @@
+/datum/armament_entry/company_import/kz_frontier
+	category = KZ_EQUIPMENT_NAME
+	company_bitflag = CARGO_COMPANY_KZ_EQUIPMENT
+
+/datum/armament_entry/company_import/kz_frontier/manufacturing
+	subcategory = "Manufacturing Equipment"
+
+/datum/armament_entry/company_import/kz_frontier/manufacturing/rapid_construction_fabricator
+	item_type = /obj/item/flatpacked_machine
+	cost = CARGO_CRATE_VALUE * 8
+
+// Tools that you could use the rapid fabricator for, but you're too lazy to actually do that
+
+/datum/armament_entry/company_import/kz_frontier/basic
+	subcategory = "Hand-Held Equipment"
+	cost = PAYCHECK_COMMAND
+
+/datum/armament_entry/company_import/kz_frontier/basic/omni_drill
+	item_type = /obj/item/screwdriver/omni_drill
+
+/datum/armament_entry/company_import/kz_frontier/basic/prybar
+	item_type = /obj/item/crowbar/large/doorforcer
+
+/datum/armament_entry/company_import/kz_frontier/basic/arc_welder
+	item_type = /obj/item/weldingtool/electric/arc_welder
+
+/datum/armament_entry/company_import/kz_frontier/basic/compact_drill
+	item_type = /obj/item/pickaxe/drill/compact
+
+// Various smaller appliances than the deployable machines below
+
+/datum/armament_entry/company_import/kz_frontier/appliances
+	subcategory = "Appliances"
+	cost = PAYCHECK_CREW
+
+/datum/armament_entry/company_import/kz_frontier/appliances/charger
+	item_type = /obj/item/wallframe/cell_charger_multi
+
+// Flatpacked, ready to deploy machines
+
+/datum/armament_entry/company_import/kz_frontier/deployables
+	subcategory = "Deployable Equipment"
+
+/datum/armament_entry/company_import/kz_frontier/deployables/solar
+	item_type = /obj/item/flatpacked_machine/solar
+	cost = PAYCHECK_CREW
+
+/datum/armament_entry/company_import/kz_frontier/deployables/solar_tracker
+	item_type = /obj/item/flatpacked_machine/solar_tracker
+	cost = PAYCHECK_CREW
+
+/datum/armament_entry/company_import/kz_frontier/deployables/station_battery
+	item_type = /obj/item/flatpacked_machine/station_battery
+	cost = PAYCHECK_COMMAND
```

Comment:
Might as well put that on the base `deployables` subtype, and just have the two with the `cost = PAYCHECK_CREW` above to keep their explicit cost declared like they do it right now.